src/shared/guards/auth.guard.ts

```diff
@@ -0,0 +1,11 @@
+import { inject } from '@angular/core';
+import { ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
+import { AuthService } from '../services/auth.service';
+
+export const isLoggedInGuard = (
```

Comment:
Is this working as a guard? Check with the already existing auth guard in the app - https://github.com/Getsov/Motorsport-Events-App/blob/dev/src/shared/guards/auth.guard.ts